tests/integration_test.rs

```diff
@@ -0,0 +1,133 @@
+extern crate addr2line;
+extern crate itertools;
+extern crate glob;
+
+use std::env;
+use std::path;
+use std::process;
+use std::io::prelude::*;
+use itertools::Itertools;
+
+#[test]
+fn identity_map() {
+    let target = env::current_exe().unwrap();
+    let mut debug = target.clone();
+
+    // point us at the real macOS debugging files
```

Comment:
Nitpick: Can you capitalize + punctuate this comment and the others below, so they're proper sentences?